README.md

```diff
@@ -104,7 +104,7 @@ You can also control the timeout values for the Golem task and subtasks (by defa
 are set to 10 minutes each) which can be adjusted as follows
 
 ```
-g_flite --task_timeout 00:10:00 --subtask_timeout 00:10:00 some_text_input.txt some_speech_output.wav
```

Comment:
Hmm, how come this change? Is it to emphasise the change? If so, is that really necessary?